src/components/manage/Blocks/Listing/ListingStyle.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { blocks } from '~/config';
+import { compose } from 'redux';
+import withObjectBrowser from '@plone/volto/components/manage/Sidebar/ObjectBrowser';
+import TemplateWidget from './TemplateWidget';
+
+const ListingStyle = ({ data, block, onChangeBlock, required = false }) => {
+  const templatesConfig = blocks?.blocksConfig?.listing?.templates;
```

Comment:
Same as before, if we remove the ``?.`` there, we should remove it here as well.